pipeline/extract/sidewalk_cafe.sql

```diff
@@ -10,3 +10,4 @@ SELECT
     latitude,
     longitude
 WHERE issued_date <= "2021-07-15"
+LIMIT 100000
```

Comment:
This was a tricky point about the data portal that was not documented: If the query has no limit, by default it will return up to 1000 rows. If you have more than that, you need to explicitly set a very large limit.